.github/workflows/validate_and_publish_release.yml

```diff
@@ -128,7 +128,7 @@ jobs:
           fetch-depth: 0
 
       - name: Updating latest-snapshot branch
-        uses: ad-m/github-push-action@40bf560936a8022e68a3c00e7d2abefaf01305a6 # v0.6.0
+        uses: ad-m/github-push-action@v0.8.0
```

Comment:
I haven't tested all of these workflows, so we should monitor after merging to see if there are any issues.

Reply:
This broke the release publishing action: https://github.com/iree-org/iree/actions/runs/9677626043

> [linux-x86_64 :: Build py-runtime-pkg Package](https://github.com/iree-org/iree/actions/runs/9677626043/job/26699686864#step:16:16)
Failed to CreateArtifact: Received non-retryable error: Failed request: (409) Conflict: an artifact with this name already exists on the workflow run